paddle/fluid/operators/math/sampler.cc

```diff
@@ -60,75 +60,30 @@ float LogUniformSampler::Probability(int64_t value) const {
   return (log((value + 2.0) / (value + 1.0))) / log_range_;
 }
 
-CustomSampler::CustomSampler(int64_t range, const float* probabilities,
+CustomSampler::CustomSampler(int64_t range, const float *probabilities,
+                             const int *alias, const float *alias_probabilities,
```

Comment:
This is a very strange API!

Will someone change alias_probabilities somewhere and affect the sampler?

Reply:
alias_probabilities will only be used in nce_op, If someone changes alias_probabilities somewhere, the sampler will be affected.